tests/conftest.py

```diff
@@ -93,3 +94,26 @@ def tarsier(credentials):
             raise ValueError("Invalid OCR provider")
 
     yield Tarsier(ocr_service)
+
+
+@asynccontextmanager
+async def page_context(html_file: str):
+    """Context manager for opening and navigating to a file URL using Playwright."""
+    async with async_playwright() as p:
+        browser = await p.chromium.launch(headless=IN_GITHUB_ACTIONS)
+        page = await browser.new_page()
+        try:
+            # Construct the absolute file path
+            html_file_path = os.path.abspath(
+                os.path.join(os.path.dirname(__file__), "mock_html", html_file)
+            )
```

Comment:
would put "mock_html" into a variable called test_html_folder or something so its more apparent what it is